libnavui-dropin/src/main/res/drawable/mapbox_map_pin_hole.xml

```diff
@@ -0,0 +1,11 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
That's what's in Figma. Map pin has a "white" colored pin hole while Trip Progress pin has transparent one. This one is the latter one. I'll remove it since we are not implementing Figma Trip Progress designs.